homeassistant/components/switchbot_cloud/strings.json

```diff
@@ -16,5 +16,33 @@
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "services": {
+    "list_devices": {
+      "name": "List devices",
+      "description": "List all devices registered to the Switchbot account."
+    },
+    "send_command": {
+      "name": "Send command",
+      "description": "Send control command to device as specified in https://github.com/OpenWonderLabs/SwitchBotAPI?tab=readme-ov-file#send-device-control-commands",
```

Comment:
Yep, looking at this list, these could all be entities, yielding a way better user experience